pytest_odoo.py

```diff
@@ -211,7 +211,7 @@ def disable_odoo_test_retry():
     try:
         from odoo.tests import BaseCase
         if hasattr(BaseCase, "run"):
-            del BaseCase.run
+            BaseCase.run = lambda *args, **kwargs: None 
```

Comment:
You could even do this without hasattr
```suggestion
        BaseCase.run = lambda *args, **kwargs: None 
```